Dockerfile

```diff
@@ -0,0 +1,39 @@
+# We need this to use GPUs inside the container
+FROM nvidia/cuda:11.2.2-base
+# Using a multi-stage build simplifies the s3prl installation
+# TODO: Find a slimmer base image that also "just works"
+FROM tiangolo/uvicorn-gunicorn:python3.8
+
+RUN apt-get update --fix-missing && apt-get install -y wget \
+    libsndfile1 \
+    sox \
+    git \
+    git-lfs
+
+RUN python -m pip install --upgrade pip
+RUN python -m pip --no-cache-dir install fairseq@git+https://github.com//pytorch/fairseq.git@f2146bdc7abf293186de9449bfa2272775e39e1d#egg=fairseq
+# TODO: Update URL if we merge back to s3prl repository
+RUN python -m pip --no-cache-dir install git+https://github.com/huggingface/s3prl.git@huggingface2#egg=s3prl
+
+COPY s3prl/ /app/s3prl
+COPY src/ /app/src
+
+# Setup filesystem
+RUN mkdir /app/data
+
+# Configure Git
+# TODO: Create a dedicated SUPERB account for the project?
+RUN git config --global user.email "lewis@huggingface.co"
+RUN git config --global user.name "SUPERB Admin"
+
+# Default args for fine-tuning
+ENV upstream_model osanseviero/hubert_base
+ENV downstream_task asr
+ENV hub huggingface
+ENV push_to_hf_hub True
+
+WORKDIR /app/s3prl
+# Each task's config.yaml is used to set all the training parameters.
+# The results of each training run are stored in /app/s3prl/result/downstream/{downstream_task}
+# and pushed to the Hugging Face Hub with name {hf_username}/superb-s3prl-{upstream_model}-{downstream_task}-uuid
+CMD python run_downstream.py -n ${downstream_task} -m train -u ${upstream_model} -d ${downstream_task} --hub ${hub} --push_to_hf_hub ${push_to_hf_hub}
```

Comment:
Could we also support the `-o` override function? Since we need to customized the training hypers through command line (without direct access to the configs inside the docker image), and the value for override argument is just a string which I think can also be passed by `-e` for the docker run command. Do @lewtun and @osanseviero think this will be nice?

Reply:
Hi there. I'm not sure I fully understand this use case in the docker image. My understanding was that we want the same hyperparameters across all models getting fine-tuned in order to be consistent.

Reply:
Yes! We want the `same set` of hypers across all models for finetuning. Like the same set of learning rate to sweep since different representations could have different preference of lr. In this use case I guess we will need to enable changing hypers through command line. Am I making any sense?

Reply:
Yes, this makes sense to me :) I'll wait for @lewtun to chime in, but this sounds good!

Reply:
yes, this is a good idea @leo19941227 ! i'll add it next week when i'm back from vacation :)